plugins/hosts/linux/cap/os_arch.rb

```diff
@@ -0,0 +1,20 @@
+include Vagrant::Util
+
+module VagrantPlugins
```

Comment:
I couldn't find the built-in method to get the architecture info. hence implementing my own here which will be used in multiple places.